<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>1065</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TOPS - Tools for Optimization and Plotting in Systems Engineering</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Interface</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_dataEdit">
            <property name="text">
             <string>Edit Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_dataPlot">
            <property name="text">
             <string>Plot Data</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_plot_system1">
          <attribute name="title">
           <string>PLOT: System 1</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>19</x>
             <y>19</y>
             <width>471</width>
             <height>481</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="plot_results_system1"/>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_table_system1">
          <attribute name="title">
           <string>TABLE: System 1</string>
          </attribute>
          <widget class="QTableWidget" name="table_results_system1">
           <property name="geometry">
            <rect>
             <x>15</x>
             <y>11</y>
             <width>1101</width>
             <height>571</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>Style</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuStyle"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Optimizer">
   <property name="text">
    <string>New System</string>
   </property>
  </action>
  <action name="actionNew_DED">
   <property name="text">
    <string>New Plot Data</string>
   </property>
  </action>
  <action name="actionLoad_REPS">
   <property name="text">
    <string>Load REPS</string>
   </property>
  </action>
  <action name="actionLoad_DED">
   <property name="text">
    <string>Load DED</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
